index.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/3da71c9faf.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="./style/main.css">
+    <script src="./javascript/main.js" defer></script>
+    <title>Global Drummers Workshop</title>
+</head>
+
+<body class="lato-font">
+    <header id="nav-container">
+        <a class="logo-container" href="/index.html">
+            <img class="logo" src="./assets/icons/Drum-festival-logo.svg" alt="Drum Festival Logo">
+        </a>
+        <nav class="navbar">
+            <ul class="navlinks-wrapper">
+                <li>
+                    <a href="./html/about.html">About</a>
+                </li>
+                <li>
+                    <a href="#programs">Program</a>
+                </li>
+                <li>
+                    <a href="#Sponsors">Sponsor</a>
+                </li>
+                <li>
+                    <a href="#performing-artist">News</a>
+                </li>
+            </ul>
+            <button type="button" class="nav-join-btn">
+                Join Drum Festival
+            </button>
+            <div class="hamburger-btn">
+                <img class="hamburger" src="./assets/icons/menu-icon.png" alt="hamburger icon">
+            </div>
```

Comment:
- [ ] Kindly make sure your hamburger menu is on the left side, currently its on the right , also remove the logo to match the design given.

Reply:
- [ ] This issue is well resolved :heavy_check_mark: